modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -316,32 +316,38 @@ def test_merge(test_data, test_data2):
 )
 @pytest.mark.parametrize("na_position", ["first", "last"], ids=["first", "last"])
 def test_sort_index(axis, ascending, na_position):
-    data = test_data["float_nan_data"]
-    modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
-
-    # Change index value so sorting will actually make a difference
-    if axis == 0:
-        length = len(modin_df.index)
+    # pandas raises the exception when `ascending=None` (pandas issue 39434).
```

Comment:
For this one, I would rather just raise the error that pandas does temporarily until the issue is fixed. Changing the test is probably not what we want.

Reply:
Okay, fixed.